extensions/wikia/Flags/specials/scripts/FlagEditForm.js

```diff
@@ -13,7 +13,7 @@ define ('ext.wikia.Flags.FlagEditForm',
 			$.when(getFormResources()).done(function (formResources) {
 				setupForm(formResources);
 
-				if (prefillData == null || prefillData.values == null) {
+				if (prefillData == null) {
```

Comment:
So explicit is better than implicit.
Use undefined check in statement or write a comment explaining intention.